api/traceid.yaml

```diff
@@ -0,0 +1,14 @@
+openapi: 3.0.0
+components:
+  schemas:
+    TRACEIDRequest:
```

Comment:
If these objects are the same should we keep the separated?

Reply:
I was thinking that probably we want to include extra data in the response. Not really needed for now.